test/dotnet/install_aspnetcore_runtime_only.sh

```diff
@@ -18,10 +18,6 @@ expected=$(fetch_latest_version "aspnetcore")
 check "Latest ASP.NET Core Runtime version installed" \
 is_aspnetcore_runtime_version_installed "$expected"
 
-# Expect this check to fail in November 2023 when .NET 8.0 becomes GA
-check "It is a flavor of .NET 7.0" \
```

Comment:
Not super confident if removing checks is the right way to go, we can also update it to .Net 8.0 (which would again fail in Nov 2024)

Appreciate any thoughts!

Reply:
I think it's ok to remove